index.html

```diff
@@ -391,6 +391,10 @@ <h2 class="text-colored">How can I apply?</h2>
 							<a href="#ticket">Click here!</a>
 						</p>
 					</div>
+					<div class="faq-question">
+						<h2 class="text-colored">Does HackHPI help with travel reimbursements?</h2>
+						<p>Unfortunately, we're not able to provide travel reimbursements.</p>
```

Comment:
maybe add `at the moment` or something alike?
We might be able to provide that in the future.